lesson_18/hw_18.py

```diff
@@ -0,0 +1,85 @@
+import requests
+import json
+
+
+class Pet:
+
+    URL = "https://petstore.swagger.io/v2/pet/"
+
+    def __init__(self, body):
+        self.body = body
+        self.id = str(body["id"])
+
+    def create_pet(self):
+        """
+        Creates a new pet then returns response list status code, reason
+        and json response body with pet params
+        """
+        response = requests.post(url=Pet.URL, json=self.body)
+        try:
+            response_body = response.json()
+            return [response.status_code, response.reason], response_body
+        except json.decoder.JSONDecodeError:
+            print("Broken json body received")
+
+    def get_pet(self):
+        """
+        Gets the pet by id then returns response list status code, reason
+        and json response body with pet params
+        """
+        response = requests.get(url=Pet.URL + self.id)
+        try:
+            response_body = response.json()
+            return [response.status_code, response.reason], response_body
+        except json.decoder.JSONDecodeError:
+            print("Broken json body received")
+
+    def delete_pet(self):
+        """
+        Deletes the pet by id then returns response list status code, reason
+        and json server response body
+        """
+        response = requests.delete(url=Pet.URL + self.id)
```

Comment:
you have a lot of similar code that you can move to some method and call it in each call